packages/appstore/src/components/modals/real-wallets-upgrade/components/content-with-link/content-with-link.scss

```diff
@@ -0,0 +1,94 @@
+.content-with-link {
+    height: auto;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+
+    @include mobile {
+        flex-direction: column-reverse;
+    }
+
+    &--has-fork {
+        height: 100%;
+    }
+
+    &--right-forked {
+        flex-direction: row;
+
+        @include mobile {
+            flex-direction: column;
+        }
+    }
+
+    &__content {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        gap: 1.6rem;
+        margin: 0rem;
+        margin-left: 1.6rem;
+
+        @include mobile {
+            gap: 0.8rem;
+            margin: 0;
+            margin-top: 1.2rem;
```

Comment:
```suggestion
            margin: 1.2rem 0rem 0rem 0rem;
```

Reply:
[Same](https://github.com/binary-com/deriv-app/pull/10145#discussion_r1335661124.)

Reply:
this one actually you can make it even shorter:
`margin: 1.2rem 0rem 0rem;`
(top, bottom, left/right)